os/net/netmgr/netdev_lwip.c

```diff
@@ -782,7 +782,7 @@ static int lwip_init_nic(struct netdev *dev, struct nic_config *config)
 		return -1;
 	}
 
-	char *rnetif = (char *)kmm_malloc(sizeof(struct netif) + sizeof(struct netdev *));
+	char *rnetif = (char *)kmm_calloc(1, sizeof(struct netif) + sizeof(struct netdev *));
```

Comment:
If you only want to initialize the memory after allocation, how about using `kmm_zalloc` instead of `kmm_calloc`?
`kmm_calloc` also initializes the memory, but it has more depth for allocation, so `kmm_zalloc` is better, I think.